community-building-program.md

```diff
@@ -0,0 +1,32 @@
+## Introduction
+In order to help users quickly integrate into the community, accelerate implementation, and jointly create a prosperous community ecology, the Volcano community is now launching the Volcano community co-construction plan. Through the Volcano community co-construction plan, you will receive support such as technical guidance, publicity and promotion, and opportunities for online/offline technical evangelism and sharing! If your company or organization recognizes Volcano's technical route, hopes to get help in the process of using Volcano, and intends to build technical influence with the Volcano community, please consider joining the program.
+
+## Joining the program will receive
+- Free training opportunities from the community:
+    - Open source community basic knowledge training, including community governance structure introduction, community development model introduction, open source contribution guidance, etc.
+    - Volcano advanced knowledge training, including architecture and function introduction, analysis of the latest version, etc.
+- Offline Communication Opportunities:
+    - Special communication for users' private scenes, directly attacking problems and protecting project privacy
```

Comment:
Communication and analysis against users' special scenario

Reply:
done